Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -229,7 +229,6 @@
                 case KEY.ESC:
                     // re-motion: block event bubbling
                     event.stopPropagation();
-                    clearTimeout(timeout);

```

Comment:
This is covered by 
https://github.com/re-motion/Trunk-SVN2Git/blob/8dd67a6ad3a3b7e38bb6f528eb1e971fef64ad9e/Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js#L168

I could not find any scenario where this line would be needed.

Reply:
Hmm.... I think it should be a separate issue (task), just to document this stuff. Otherwise, autocomplete will kill us with special case research.